src/Uno.UI/UI/Xaml/Input/ManipulationStartingRoutedEventArgs.cs

```diff
@@ -0,0 +1,22 @@
+using Windows.UI.Input;
+using Uno.UI.Xaml.Input;
+
+namespace Windows.UI.Xaml.Input
+{
+	public partial class ManipulationStartingRoutedEventArgs : RoutedEventArgs, ICancellableRoutedEventArgs
+	{
+		public ManipulationStartingRoutedEventArgs() { }
+
+		internal ManipulationStartingRoutedEventArgs(UIElement container)
+			: base(container)
+		{
+			Container = container;
+			Mode = container.ManipulationMode;
```

Comment:
That's a strange place to extract this information from the container.

Reply:
Indeed, but I've to capture it in the ctor as it can be customized in this event handler. I've added few comments to explain why I capture it here from the `container` (while I should get it from the inner args).